library/common/android_jni_utils.cc

```diff
@@ -0,0 +1,21 @@
+#include <android/log.h>
+#include <ares.h>
+#include <jni.h>
+
+// NOLINT(namespace-envoy)
+
+jint init_jvm(JavaVM* static_jvm, jobject connectivity_manager) {
+  // See note above about c-ares.
```

Comment:
This comment is no longer applicable (it referred to the remainder of the comments below).